### ISSUE TITLE
Feature Request: Conversion between transit/edn/JSON

### ISSUE THREAD

**Issue by riotrah** [BetterThanTomorrow/calva#2222]
Would it be possible to extend the existing conversion features to include translation between transit/edn/json - ie, somehow include [borkdude/jet](https://github.com/borkdude/jet) or its feature set into calva?

**Comment by PEZ** [comment]
Certainly. Best would be if we could use jet as a ClojureScript library, I think. Not sure if that's possible/hard/easy to do.

**Comment by borkdude** [comment]
jet is not implemented as a CLJS library, it's designed as a command line tool. If the feature is "transform this file to EDN / transit / JSON" I think it's best to implement that as a bespoke feature in Calva, or write a joyride script which shells out to jet.
